src/components/MapEditor/__tests__/MapInteractive.test.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+
+import Map from '..';
+import MAP_OPTIONS from '../../../shared/services/configuration/map-options';
+
+describe('components/MapInteractive', () => {
+  const testLocation = {
+    geometrie: {
+      type: 'Point',
+      coordinates: [4, 52],
+    },
+  };
+
+  it('should render the map', () => {
+    const { container, getByText } = render(<Map options={MAP_OPTIONS} location={{}}/>);
+
+    // Map
+    expect(getByText('Leaflet')).toBeInTheDocument();
```

Comment:
Where does this text come from? I don't see any reference of it in the `MapEditor` component. We cannot rely on external dependencies in our test cases.

Reply:
Sure thing! I've added a test-id for this